htm/src/nupic/types/Serializable.hpp

```diff
@@ -50,8 +50,8 @@ class Serializable {
   virtual inline int getSerializableVersion() const { return SERIALIZABLE_VERSION; }
 
   virtual inline void saveToFile(std::string filePath) const {
-      std::string dirPath = Path::getParent(filePath);
-	  Directory::create(dirPath, true, true);
+      //std::string dirPath = Path::getParent(filePath);
+	  //Directory::create(dirPath, true, true);
```

Comment:
> is this needed?

The Path:: and Directory:: things here were just to automatically create the parent directories if they don't already exist.  Not sure that was a good idea.  The caller should make sure that the directory into which it is writing already exists.  I commented it out in case we change our mind but that would be a dependency on the NetworkAPI.

Reply:
> The caller should make sure that the directory into which it is writing already exists. I commented it out in case we change our mind but that would be a dependency on the NetworkAPI.

:+1: even remove the calls and the classes, if possible